cross/x86/toolchain.cmake

```diff
@@ -7,6 +7,7 @@ set(CMAKE_SYSTEM_PROCESSOR i686)
 add_compile_options("-m32")
 add_compile_options("--sysroot=${CROSS_ROOTFS}")
 add_compile_options("-Wno-error=unused-command-line-argument")
+add_compile_options("-Wno-missing-prototype-for-cc")
```

Comment:
Where was the warning reported? I would prefer fixing it instead of hiding it unless it happens on too many places.